README.md

```diff
@@ -97,7 +102,34 @@ The SDO owner services are packaged as a single docker container that can be run
    ```bash
    curl -sS -w "%{http_code}" -X POST $SDO_RV_URL/mp/113/msg/20 | jq
    ```
+   
+### <a name="gen-keypair"></a>Generate Owner Key Pairs
+
+If you want to expedite the process of creating key pairs, you can run `keys/generate-key-pair.sh`
+To run this script you must be using Ubuntu. 
+
+1. Go to the directory where you want your generated keys to be saved then download `generate-key-pair.sh`, which is used to create key pairs for Owner Attestation:
+
+   ```bash
+   curl -sSLO https://raw.githubusercontent.com/open-horizon/SDO-support/master/keys/generate-key-pair.sh
+   chmod +x generate-key-pair.sh
+   ```
+   
+2. Run `generate-key-pair.sh` script. If this process is being ran for a non production environment you **do not** need to create your own keys. 
```

Comment:
Remove: "If this process is being ran for a non production environment you **do not** need to create your own keys."  Because if they are using the default sample keys, they can skip the whole section, not just this step.